src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+the completion time for as long as the animation runs. We believe it is worthwhile to investigate
+options on how to use heuristics to capture work that is asynchronous and which should be completed
+ASAP. However, we want to be really careful when doing so because we don’t want to penalize work
+that is meant to take a long time to be finished. Thus, our initial effort will look at step 5 as
+the end point: it will only consider synchronous work and the amount of time it takes to paint after
+such work is completed. That is, we’re not going to apply heuristics to guess the work that would be
+kicked off asynchronously in step 4 in our initial effort.
+
+It’s worth noting that, in many cases, work should be executed synchronously. In fact, this may be
+unavoidable because events are sometimes dispatched one after the other and the event handlers need
+to be executed in order. That said, we will still miss important work, like events which trigger
+fetching or which rely on important work to be done at the next `requestAnimationFrame` callback,
+for example. {% endDetails %}
+
+### Group events into interactions
+
+Extending the metric measurement from _delay_ to _duration_ is a good first step, but it still
+leaves a critical gap in the metric: it focuses on individual events and not the user experience of
+interacting with the page.
+
+Many different events can fire as a result of a single user interaction, and separately measuring
+each doesn't build a clear picture of what the user experiences. We want to make sure our metric
+captures the full amount of time a user has to wait for a response when tapping, pressing keys,
+scrolling, and dragging as accurately as possible. So we're introducing the concept of
+***interactions*** to measure the latency of each.
+
+#### Interaction types
+
+The following table lists the four interactions we want to define along with the DOM events that
+they're associated with. Note that this is not quite the same as the set of all events that are
+dispatched when such user interaction occurs. For instance, when a user scrolls, a scroll event is
+dispatched, but it happens after the screen has been updated to reflect the scrolling, so we don't
+consider it part of the interaction latency.
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Interaction</th>
+        <th>Start / end</th>
+        <th>Desktop events</th>
+        <th>Mobile events</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td rowspan="3">Keyboard</td>
+        <td rowspan="2">Key pressed</td>
+        <td><code>keydown</code></td>
+        <td><code>keydown</code></td>
+      </tr>
+      <tr>
+        <td><code>keypress</code></td>
+        <td><code>keypress</code></td>
+      </tr>
+      <tr>
+        <td>Key released</td>
+        <td><code>keyup</code></td>
+        <td><code>keyup</code></td>
+      </tr>
+      <tr>
+        <td rowspan="7">Tap / drag</td>
```

Comment:
Nit: `Tap / Drag` feel like this is a single interaction type with alternative names, or two steps.

I think this is meant to be two logical rows in the table, which happen to have the same events?
Would it be too much effort to make it two rows?

Reply:
...looking at the full table, it's probably too much effort.

Still, I would consider doing something like `<b>Tap</b> or, <b>Drag</b>`